src/frontend/src/pages/Account.tsx

```diff
@@ -86,64 +103,163 @@ function openFieldModal(field: Field) {
   function parseUser(json: any) {
     let user: User = {
       id: json.id,
-      firstName: json.firstName? json.firstName : "FirstName",
-      lastName: json.lastName? json.lastName: "LastName",
+      firstName: json.firstName ? json.firstName : 'FirstName',
+      lastName: json.lastName ? json.lastName : 'LastName',
       userName: json.email,
       eMail: json.email,
       balance: Number.parseFloat(json.balance),
-      accountType: json.user_permissions.length === 0? 'user' : 'employee',
+      accountType: json.user_permissions.length === 0 ? 'employee' : 'employee',
       hours: 4,
+    };
+
+    return user;
+  }
+
+  function postName(field1: string, field2: string, password: string) {
```

Comment:
Are you not able to make a single POST request that contains all fields in one? I.e. post a body that contains firstName, lastName, email, password? Just asking for my own curiousity since I haven't seen how any of the user stuff works

Reply:
If you're referring to the fact that I have multiple functions for posting name/username/email, there's slightly different input verification logic between them, could totally factor out the fetch request into a separate function though. Also at this point I'm not sure if I'm doing a post or a put because I can't get either to work with our endpoint.